frontend/src/travelTimeQuery.js

```diff
@@ -78,8 +78,21 @@ export class TravelTimeQuery {
             this.holidaysAreRelevant ? this.#holidayOption.holidaysIncluded : 'NA'
         )
         record.set('hoursInRange', this.hoursInRange)
-        record.set('mean_travel_time_minutes', this.#results?.travel_time?.minutes)
-        record.set('mean_travel_time_seconds', this.#results?.travel_time?.seconds)
+        // get the standard deviation of the travel time observations
+        // start by calculating the mean again, separately
+        // this is because the tt means we use aren't straight averages, for historical reasons
+        let sum_obs = this.#results.observations.reduce((cumsum, tt) => {
+            return cumsum + tt.seconds
+        }, 0)
+        let mean_obs = sum_obs / this.#results.observations.length
+        record.set('mean_travel_time_seconds', mean_obs)
+        // sum of the squared deviations from the mean
+        let sum_sq_dev = this.#results.observations.reduce((cumsum, tt) => {
+            return cumsum + (mean_obs - tt.seconds)**2
+        }, 0)
+        let variance = sum_sq_dev / this.#results.observations.length
```

Comment:
Should we use n-1 for standard deviation of a sample?
![Standard-Deviation-Formula](https://github.com/user-attachments/assets/6e019c75-68a7-495a-a85c-b6b46bbac270)